cst438_register/src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -30,7 +30,7 @@ public class StudentController {
 	
 	@PostMapping("/student/add")
 	@Transactional
-	public StudentDTO addCourse( @RequestBody StudentDTO studentDTO  ) { 
+	public StudentDTO addStudent( @RequestBody StudentDTO studentDTO  ) { 

```

Comment:
How come the method name change?

Reply:
Just a forgotten artifact from copy pasta, didn't notice it until now.